src/pages/app-resources/app-resources.tsx

```diff
@@ -121,6 +121,10 @@ export class AppResources {
       const form = document.getElementById('top');
       form.scrollIntoView({ block: 'start', behavior: 'smooth' });
 
+      //Ensure this downloads on firefox
+      if (navigator.userAgent.indexOf('Firefox') > 0) {
+        window.location.href = '/assets/PWA%20White%20Paper.pdf';
+      }
```

Comment:
If its on Firefox, we need to changes the window location, nice, but should we add a if to only create the anchor element when not on Firefox?